packages/component/src/Utils/Fade.js

```diff
@@ -0,0 +1,28 @@
+import { useLayoutEffect, useState } from 'react';
```

Comment:
Should add a short comment that this is for a11y/live region

Reply:
```js
// Initially, <Fade> will render all of its children.
// After a period of time (specified through the "fadeAfter" prop), <Fade> will stop rendering its children and removing it from DOM.
```